src/components/clusterConfiguration/HostInventory.tsx

```diff
@@ -32,12 +32,21 @@ const HostRequirementsContent = () => {
   const workerRam = fileSize((worker?.ramMib || 8 * 1024) * 1024 * 1024, 2, 'iec');
 
   return (
-    <Text component="p">
-      Three master hosts are required with at least {master?.cpuCores || 4} CPU cores, {masterRam}{' '}
-      of RAM, and {master?.diskSizeGb || 120} GB of filesystem storage each. Two or more additional
-      worker hosts are recommended with at least {worker?.cpuCores || 2} CPU cores, {workerRam} of
-      RAM, and {worker?.diskSizeGb || 120} GB of filesystem storage each.
-    </Text>
+    <List>
+      <ListItem>
+        Supervisors: At least {master?.cpuCores || 4} CPU cores, {masterRam} RAM,{' '}
```

Comment:
Shouldn't the rename from 'master' to 'supervisor' be done in a single focused change in cooperation with the backend? This will potentially confuse users for a long time until we remove all uses of 'master'

Reply:
I'm just following the mockup, does it matter that much if we leave this like it is right now?

Reply:
Reverted that to "Masters"